### DISCUSSION TITLE
How to set proxy for reuqest?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#16592] [Questions]
I know that nuxt uses the ohmyfetch library... is there any way to set the proxy like it's done in axios?
https://masteringjs.io/tutorials/axios/proxy

**Comment by sevillaarvin** [comment]
I don’t think this is supported in ohmyfetch. Perhaps file it as a feature request?

**Comment by ironytr** [comment]
and none of the proxy modules working with latests versions of nuxt3...
